pipe/pipe.py

```diff
@@ -63,6 +77,10 @@ def run_owasp_check(self):
         if self.disableOssIndex == True:
             owasp_command.append('--disableOssIndex')
 
+        if self.extra_args:
+            warnings.warn(f"Dependency check has been extended: {self.extra_args}", UserWarning)
```

Comment:
Do we need a warning here? :thinking:

Reply:
I think we do need some indication if we should extend the dependency checker.

Reply:
I'm not sure what you mean by extending the dependency checker?